lib/honeydew.ex

```diff
@@ -99,6 +134,20 @@ defmodule Honeydew do
     %{queue: Map.delete(queue_status, :monitors), workers: workers}
   end
 
+  @doc """
+  Filters the jobs currently on the queue.
+
+  ## Examples
+
+  Filter jobs with a specific task.
+
+      Honeydew.filter(:my_queue, &match?(%Honeydew.Job{task: {:ping, _}}, &1))
+
+  Return all jobs.
+
+      Honeydew.filter(:my_queue, fn _ -> true end)
+  """
```

Comment:
can you add a note that this doesn't return a stream, but a list, and could be quite memory intensive?

Reply:
Absolutely 👍